cockroachdb/templates/ingress.yaml

```diff
@@ -40,11 +40,11 @@ spec:
         paths:
   {{- range $path := $paths }}
           - path: {{ $path }}
-            {{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+            {{- if semverCompare ">=1.19-0" $.Capabilities.KubeVersion.GitVersion -}}
```

Comment:
can you please move this to Version instead of GitVersion? like `$.Capabilities.KubeVersion.Version`

Reply:
I guess even better if I use:
```
{{ if $.Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
```

So far I just copied what I saw elsewhere in the chart, but will do an update.

Reply:
We just started making this change - thanks @JorritSalverda !!